mappings/com/mojang/blaze3d/systems/RenderSystem.mapping

```diff
@@ -58,16 +75,45 @@ CLASS com/mojang/blaze3d/systems/RenderSystem
 	METHOD colorMaterial (II)V
 		ARG 0 face
 		ARG 1 mode
+	METHOD defaultAlphaFunc ()V
+	METHOD defaultBlendFunc ()V
 	METHOD deleteTexture (I)V
 		ARG 0 texture
 	METHOD depthFunc (I)V
 		ARG 0 func
 	METHOD depthMask (Z)V
 		ARG 0 mask
+	METHOD disableAlphaTest ()V
+	METHOD disableBlend ()V
+	METHOD disableColorLogicOp ()V
+	METHOD disableColorMaterial ()V
+	METHOD disableCull ()V
+	METHOD disableDepthTest ()V
+	METHOD disableFog ()V
+	METHOD disableLighting ()V
+	METHOD disableLineOffset ()V
+	METHOD disablePolygonOffset ()V
+	METHOD disableRescaleNormal ()V
+	METHOD disableScissor ()V
+	METHOD disableTexture ()V
```

Comment:
Again here and in the fields, not sure what the point in these are as they dont have a yarn name?

Reply:
Oops sorry, don't know what happened there.
I'll fix that when I get time